src/constants/productTokens.ts

```diff
@@ -89,11 +89,13 @@ export const productTokensBySymbol = {
   'BTC2x-FLI': Bitcoin2xFlexibleLeverageIndex,
 }
 
-export default [
+const ProductTokens = [
   DefiPulseIndex,
   MetaverseIndex,
   CoinsharesCryptoGoldIndex,
   Ethereum2xFlexibleLeverageIndex,
   IndexToken,
   Bitcoin2xFlexibleLeverageIndex,
 ]
+
+export default ProductTokens
```

Comment:
It came as a warning on build to put default exports into a variable and then export